activemodel/lib/active_model/attribute_set.rb

```diff
@@ -7,7 +7,7 @@
 
 module ActiveModel
   class AttributeSet # :nodoc:
-    delegate :each_value, :fetch, :except, to: :attributes
+    delegate :each_value, :fetch, :except, :slice!, to: :attributes
```

Comment:
Speaking of `slice!`...

`delegate` kind of serves a weird role in `AttributeSet`.  It works with `each_value` and `fetch`, if you don't look too closely at the return value of `each_value`.  But for methods like `slice!`, it returns a `Hash` where I would expect another `AttributeSet` (just like I expect `ActionController::Parameters#slice!` to return an `ActionController::Parameters`).

(`except` also has this problem, but that's outside the scope of this PR. :laughing:)